### PR TITLE
Send corners of min area rectangles

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: photon-lib/src/main/native/cpp/photonlib/PhotonTrackedTarget.cpp [hunk 1]
```diff
@@ -54,6 +64,15 @@ Packet& operator>>(Packet& packet, PhotonTrackedTarget& target) {
   target.cameraToTarget =
       frc::Transform2d(frc::Translation2d(units::meter_t(x), units::meter_t(y)),
                        units::degree_t(rot));
+
+  target.corners.clear();
+  for(int i = 0; i < 4; i++) {
+    double first = 0;
+    double second = 0;
+    packet >> first >> second;
+    target.corners.push_back(std::make_pair(first, second));
```
Comment (calcmogul) [inline photon-lib/src/main/native/cpp/photonlib/PhotonTrackedTarget.cpp:73]:
```suggestion
    target.corners.emplace_back(first, second);
```

File: photon-lib/src/main/native/cpp/photonlib/SimVisionSystem.cpp [hunk 2]
```diff
@@ -86,7 +86,11 @@ void SimVisionSystem::ProcessFrame(frc::Pose2d robotPose) {
 
     if (CamCanSeeTarget(distHypot, yawAngle, pitchAngle, area)) {
       PhotonTrackedTarget newTgt = PhotonTrackedTarget(
-          yawAngle.value(), pitchAngle.value(), area, 0.0, camToTargetTrans);
+          yawAngle.value(), pitchAngle.value(), area, 0.0, camToTargetTrans,
+           {std::make_pair(1,2),
+            std::make_pair(3,4),
+            std::make_pair(5,6),
+            std::make_pair(7,8)});
```
Comment (calcmogul) [inline photon-lib/src/main/native/cpp/photonlib/SimVisionSystem.cpp:93]: Prefer `std::pair{1.0, 2.0}` over `std::make_pair(1.0, 2.0)`.

File: build.gradle [hunk 3]
```diff
@@ -44,7 +44,7 @@ spotless {
         removeUnusedImports()
         trimTrailingWhitespace()
         endWithNewline()
-    }
+ targets   }
```
Comment (calcmogul) [inline build.gradle:47]: Remove this.

**Review by calcmogul — CHANGES_REQUESTED** [review]

**Comment by calcmogul** [comment]
You need to run spotless.

File: photon-server/src/main/resources/web/index.html [hunk 4]
```diff
@@ -1 +1 @@
-<p>UI has not been copied!</p> 
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/favicon.png><title>PhotonVision</title><link href=/js/chunk-2d216214.4302abb0.js rel=prefetch><link href=/js/chunk-2d216257.0de4b8cc.js rel=prefetch><link href=/js/chunk-ad0384ec.7dafafc8.js rel=prefetch><link href=/css/app.55442d99.css rel=preload as=style><link href=/css/chunk-vendors.25378257.css rel=preload as=style><link href=/js/app.7a29a66f.js rel=preload as=script><link href=/js/chunk-vendors.7a922cb5.js rel=preload as=script><link href=/css/chunk-vendors.25378257.css rel=stylesheet><link href=/css/app.55442d99.css rel=stylesheet></head><body><noscript><strong>We're sorry but PhotonVision doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.7a922cb5.js></script><script src=/js/app.7a29a66f.js></script></body></html>
```
Comment (Bankst) [inline photon-server/src/main/resources/web/index.html:1]: bade

**Review by Bankst — CHANGES_REQUESTED** [review]

File: photon-server/src/main/resources/web/index.html (hunk 4 above)
Comment (mcm001) [reply photon-server/src/main/resources/web/index.html:1]: aaaaaaaaaaaaaaaaaa

**Review by Bankst — APPROVED** [review]

**Pull request merged**